examples/generate.sh

```diff
@@ -190,12 +205,22 @@ else
 fi
 
 # Generate cluster resources.
-kustomize build "${SOURCE_DIR}/cluster" | envsubst > "${CLUSTER_GENERATED_FILE}"
-echo "Generated ${CLUSTER_GENERATED_FILE}"
+if [[ $CONTROLPLANE == "single-node" ]]; then
```

Comment:
I would propose to call these folder cluster-single-node and cluster-multi-node then you can remove the if else and just use the $CONTROLPLANE var directly in the folder path

Reply:
+1

Reply:
Agreed, this will simplify things !!
Would it not be better to just name it "single-node" and "multi-node" instead. Because using naming "cluster-single-node" would not go well in control-plane folder. Just asking, will do it what fits best

Reply:
What do you mean with go well? The folder would be relatively long but we definitely need separate folders for control plane an cluster (or at least it would be easier if it's not mixed)

Reply:
Ah now I know what you mean. If we have a cluster and controlplane folder it would be better to not repeat cluster and controlplane. So yes you're right. For example cluster/multi-node would be better.

Reply:
Ya sure I will. :)